### ISSUE TITLE
[Fluent] `NavigationBar` is not themed

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/uno.toolkit.ui#541]
## Current behavior

In light theme it looks like this (ok)

![image](https://user-images.githubusercontent.com/1075116/230768732-0ca62c29-1d6e-4366-81ef-d6a84833609a.png)

In dark theme it looks like this (not ok)

![image](https://user-images.githubusercontent.com/1075116/230768697-9665d751-21e4-4e76-9a06-8bfd3848e331.png)

Same for the three dots button:

![threedots](https://user-images.githubusercontent.com/1075116/230769250-a44a7334-3a41-47a5-96e1-30ba52055f19.gif)

## Environment

Nuget Package:

Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renders for Xamarin.Forms
- [ ] Windows
- [ ] Build tasks

Visual Studio:

- [ ] 2017 (version: )
- [ ] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by nickrandolph** [comment]
cc @kazo0 I think this is an issue with the fluent navigationbar styling not set to handle theming correctly

**Comment by kazo0** [comment]
Yeah the Toolkit controls really don't have a lot of love given to the non-Material styles at this point so we're going to need to take time to see how these should look in a non-Material context
